tests/test_seekrpc.py

```diff
@@ -6,20 +6,26 @@
 import tempfile
 
 testDir = os.path.abspath(os.path.dirname(__file__))
+sleipnirDir = os.path.dirname(testDir)
 testInputsDir = os.path.join(testDir, 'test_inputs')
 testOutputsDir = os.path.join(testDir, 'test_outputs')
-sleipnirDir = os.path.dirname(testDir)
-sleipnirBin = os.path.join(sleipnirDir, 'Debug')
 sampleBcDir = os.path.join(testOutputsDir, 'sampleBC')
+sleipnirBin = os.path.join(sleipnirDir, 'Debug')
 seekScriptsDir = os.path.join(sleipnirDir, 'scripts', 'seek')
 sys.path.append(seekScriptsDir)
 pytoolsDir = os.path.join(testDir, 'bioinform_tests', 'pytools')
 sys.path.append(pytoolsDir)
+seekRpcDir = os.path.join(sleipnirDir, 'tools', 'SeekRPC')
+seekRpcPyDir = os.path.join(seekRpcDir, 'gen-py')
+sys.path.append(seekRpcPyDir)
 from pytestHelper import createSampleDatabase
 from rank_correlation import files_rank_correlation
+from seek_rpc import SeekRPC
```

Comment:
Here you have import from seek_rpc. We should decide just to go with symlink pyseek (see my comment below) or just use seek_rpc as it here. Now I think seek_rpc makes more sense since you don't need to care about this symlink

Reply:
Yes, good suggestion, I'll switch to using gen-py/seek_rpc/